projects-2.13.md

```diff
@@ -52,6 +52,8 @@ Add in sbt using `libraryDependencies += ...`:
     "com.github.xuwei-k"                     %% "nobox"                   % "0.3.0"
     "com.github.xuwei-k"                     %% "zeroapply"               % "0.2.2"
     "io.argonaut"                            %% "argonaut"                % "6.2.3"
+    "io.crashbox"                            %% "spray-json"              % "1.3.5-4"
```

Comment:
there's also an official release: `"io.spray" %% "spray-json" % "1.3.5"`

Reply:
349c6df143b3e155cd89e371844ecb1c45156cd8

Reply:
@Philippus I was adding `sconfig` and it uses the `jodersky/spray-json` fork as it is cross platform (Native and Scala.js) so I just added it too. The library is used for tests to compare with an independent JSON parser. I think there are only build changes between the two.